File: lab4 mpp/ProjectNetworking/VoluntarDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace teledon.networking.dto
{
	[Serializable]
	public class VoluntarDTO
    {
        private string username;
        private string password;

        public VoluntarDTO(string u, string p)
        {
            this.username = u;
            this.password = p;
        }
		public virtual string Username
		{
			get
			{
				return username;
			}
			set
			{
				this.username = value;
			}
		}


		public virtual string Passwd
		{
			get
			{
				return this.password;
			}
		}
	}


}

File: GUImpp/TeledonNetworking/src/main/java/teledon/network/dto/VoluntarDTO.java
package teledon.network.dto;

import java.io.Serializable;

public class VoluntarDTO implements Serializable {


    private String username;
    private String password;

    public VoluntarDTO(String id) {
        this(id,"");
    }

    public VoluntarDTO(String id, String passwd) {
        this.username = id;
        this.password = passwd;
    }

    public String getUsername()
    {
        return username;
    }

    public String getPassword() {
        return password;
    }

    @Override
    public String toString() {
        return "VoluntarDTO{" +
                "username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }

}

